docs/parameters.md

```diff
@@ -31,3 +31,26 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | io2 (blockExpress = false) | 100            | 64000         | 500               |
 | io2 (blockExpress = true)  | 100            | 256000        | 500               |
 | gp3                        | 3000           | 16000         | 500               |
+
+## Volume Availability Zone and Topologies
+
+The EBS CSI Driver supports the [`WaitForFirstConsumer` volume binding mode in Kubernetes](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode). When using `WaitForFirstConsumer` binding mode the volume will automatically be created in the appropriate Availability Zone and with the appropriate topology. The `WaitForFirstConsumer` binding mode is recommended whenever possible for dynamic provisioning.
+
+When using static provisioning, or if `WaitForFirstConsumer` is not suitable for a specific usecase, the Availability Zone can be specified via the standard CSI toplogy mechanisms. The EBS CSI Driver supporsts both the key `topology.kubernetes.io/zone` or the key`topology.ebs.csi.aws.com/zone` to speicify the Availability Zone.
```

Comment:
speicify -> specify

Reply:
Fixed that and also spotted a missing space on the same line that I just fixed

Reply:
supporsts  -> supports